src/runtime/thread_pool_common.h

```diff
@@ -239,6 +239,9 @@ using namespace Halide::Runtime::Internal;
 extern "C" {
 
 WEAK int halide_set_num_threads(int n) {
```

Comment:
How is an argument of zero handled in this case? It looks like it would set target_a_team_size to zero.